### PR TITLE
Breathing/Circulation - Add Device Monitoring During CPR

### PR DESCRIPTION
**Pull request by mazinskihenry**
**When merged this pull request will:**
- Allows medics performing CPR to see AEDX and PulseOx monitoring during CPR

- [ ] Add watch monitoring

### IMPORTANT

- [Development Guidelines](https://ace3.acemod.org/wiki/development/) are read, understood and applied.
- Title of this PR uses our standard template `Component - Add|Fix|Improve|Change|Make|Remove {changes}`.

### CODE REVIEW

File: addons/breathing/functions/fnc_fullHealLocal.sqf [hunk 1]
```diff
@@ -27,6 +27,7 @@ _patient setVariable [QGVAR(deepPenetratingInjury), false, true];
 _patient setVariable [QGVAR(etco2Monitor), [], true];
 _patient setVariable [QGVAR(breathRate), 15, true];
 _patient setVariable [QGVAR(nasalCannula), false, true];
+_patient setVariable [QGVAR(pulseOxAttached), false, true];
```
Comment (MiszczuZPolski) [inline addons/breathing/functions/fnc_fullHealLocal.sqf]: fullHeal does not clear attached pulseox, you might want to move it to init.sqf

File: addons/circulation/functions/fnc_CPRStart.sqf [hunk 2]
```diff
@@ -47,10 +61,60 @@ if (_notInVehicle) then {
...
+        };
+        case (_deviceCode == 0): { "CPR_MONITOR" cutText ["", "PLAIN"]; GVAR(PulseOxDisplay) = false; GVAR(CPRDisplayActive) = false;};
+    };
+}, 2, [_medic, _patient]] call CBA_fnc_addPerFrameHandler;
+
 [{
     params ["_medic", "_patient", "_notInVehicle", "_CPRStartTime"];
     
-    [LLSTRING(StopCPR), "", ""] call ACEFUNC(interaction,showMouseHint);
+    [LLSTRING(StopCPR), "Change Monitoring Device", ""] call ACEFUNC(interaction,showMouseHint);
```
Comment (MiszczuZPolski) [inline addons/circulation/functions/fnc_CPRStart.sqf]: stringtable

File: addons/circulation/functions/fnc_fullHealLocal.sqf [hunk 3]
```diff
@@ -22,6 +22,9 @@ _patient setVariable [QGVAR(cprCount), 2, true];
 _patient setVariable [QGVAR(heartRestart), false, true];
 _patient setVariable [QGVAR(cardiacArrestType), 0, true];
 
+_patient setVariable [QGVAR(deviceCode), 0, true];
```
Comment (MiszczuZPolski) [inline addons/circulation/functions/fnc_fullHealLocal.sqf]: why do we need to clear this on fullHeal? (question to both variables, i do quick review and cant check this one big function rn)

**Review by MiszczuZPolski — CHANGES_REQUESTED** [review]

File: addons/circulation/functions/fnc_fullHealLocal.sqf (hunk 3 above)
Comment (mazinskihenry) [reply addons/circulation/functions/fnc_fullHealLocal.sqf]: Removed from fullHeal

File: addons/breathing/functions/fnc_fullHealLocal.sqf (hunk 1 above)
Comment (mazinskihenry) [reply addons/breathing/functions/fnc_fullHealLocal.sqf]: Remove the variable entirely, I can do the same with the pulseoximeter variable

**Review by MiszczuZPolski — APPROVED** [review]

**Pull request merged**